public/app/core/components/AppChrome/DockedMegaMenu/MegaMenuItem.tsx

```diff
@@ -121,7 +121,9 @@ export function MegaMenuItem({ link, activeItem, level = 0, onClick }: Props) {
                 />
               ))
           ) : (
-            <div className={styles.emptyMessage}>{link.emptyMessage}</div>
+            <div className={styles.emptyMessage} aria-live="polite">
```

Comment:
idk if i'm doing something wrong, but this doesn't announce for me with voiceover? 🤔

Reply:
It should only announce when the expand/collapse button of the Starred section is clicked.

Reply:
ah nm, ignore me, got it working 🙌